packages/fxa-react/lib/AppLocalizationProvider.test.js

```diff
@@ -7,12 +7,10 @@
 import IntlPolyfill from 'intl';
 Intl.NumberFormat = IntlPolyfill.NumberFormat;
```

Comment:
Not really part of the issue for the PR, but we no longer need the polyfill as the node version we use now has that function.